pkg/alias/map.go

```diff
@@ -70,16 +70,18 @@ func (m *Map) Size() int {
 }
 
 // Series returns keys of aliases map
-func (m *Map) Series(isReverse bool) []string {
+func (m *Map) Series(isReverse bool) ([]string, []string) {
```

Comment:
It doubles memory consumption for metric names of each request. Maybe we'd try to fix the current broken place?

Reply:
Yes, double memory (for metric names only). I can try some refactor and avoid this.